plasmapy/physics/tests/test_distribution.py

```diff
@@ -42,3 +47,84 @@ def test_Maxwellian_1D():
 
     with pytest.raises(ValueError):
         Maxwellian_1D(1*u.m/u.s, T=1*u.K, particle='XXX')
+
+def test_Maxwellian_speed_1D():
+    r"""test the 1D Maxwellian speed distribution function"""
+    T = 1.0 * u.eV
+    particle = 'H'
+    # get thermal velocity and thermal velocity squared
+    vTh = thermal_speed(T, particle=particle, method="most_probable")
+    # setting up integration from 0 to 10*vTh
+    xData1D = np.arange(0, 10.01, 0.01) * vTh
+    yData1D = Maxwellian_speed_1D(v=xData1D,T=T,particle=particle)
+    # integrating, this should be close to 1
+    integ = spint.trapz(y=yData1D, x=xData1D)
+    print("1D integral {0}".format(integ))
+    exceptStr = "Integral of distribution function should be 1."
+    assert np.isclose(integ.value, 1), exceptStr
+    
+    
+def test_Maxwellian_velocity_3D():
+    r"""test the 3D Maxwellian velocity distribution function"""
+    T = 1.0 * u.eV
+    particle = 'H'
+    # get thermal velocity and thermal velocity squared
+    vTh = thermal_speed(T, particle=particle, method="most_probable")
+    # defining a closure with partially applied arguments
+    def velDist3D(vx, vy, vz):
+        velDist = Maxwellian_velocity_3D(vx=vx,
+                                         vy=vy,
+                                         vz=vz,
+                                         T=T, 
+                                         particle=particle)
+        return velDist
+    # setting up integration from -10*vTh to 10*vTh, which is close to Inf
+    infApprox = 10 * vTh
```

Comment:
I'm trying to think of a way we could check whether this is a good approximation (aside from calculating this again for 20 vTh) and raise a warning if it turns out not to be one.

Reply:
I think this ends up being something like 50 or 100 e-foldings of decay, since we have roughly `exp(-v**2 / vTh**2) => exp(-(10*vTh)**2 / vTh**2) => exp(-100)`, times some coefficients. In any case, this test works for the 1D case and passes the `isclose` to 1 test with default errors.

Reply:
... 50 e-foldings? Okay. Nevermind the warning. :laughing: